contents/articles/2017-01_render-visual-field/index.md

```diff
@@ -0,0 +1,22 @@
+--- 
+title: Adapt Rendering to the User's Current Visual Field
+date: Mon Jan 09 2017 17:13:14 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Daniel Pohl et al., propose to use the user's visual field of the eye gaze for render optimizations and increase performance..." 
+featured_img: "../../../../media/images/blog/visfield.png"
+featured_img_thumb: "../../../../media/images/blog/thumb/visfield.png" 
+---
+
+<img src="../../../../media/images/blog/visfield.png" class='Feature-image u-padBottom--1' alt="Adapt Rendering to the User's Current Visual Field">
+Image Source: [Concept for Using Eye Tracking in a Head-Mounted Display to Adapt Rendering to the User's Current Visual Field](https://perceptual.mpi-inf.mpg.de/wp-content/blogs.dir/12/files/2016/11/pohl2016_vrst.pdf)
+
+<br>
+Advances in head-mounted displays for further immersion increases the need for performance optimizations to handle rendering workload. [Daniel Pohl](http://www.intel.com/content/www/us/en/research/people/intel-labs-bio-daniel-pohl.html) et al., propose to use the user's visual field of the eye gaze for render optimizations and increase performance.
+
+The research shows that there are lens defect in the current generation of HMD's where certain parts of the screen are not visible. Daniel Pohl et al., have provided a calibraton routine using Pupil Labs's [Oculus Rift DK2 add-on cup](https://pupil-labs.com/store/#vr-ar) in Oculus DK2 head-mounted display to measure these lens effects to find the visual field.
```

Comment:
Edit:

The researchers have developed a calibration routine using Pupil Labs' [Oculus Rift DK2 add-on cup](https://pupil-labs.com/store/#vr-ar. Their calibration routine enables one to determine an individual user's visual field. The calibration demonstrates that users can actually see more of the VR environment when fixating on the center of a calibration area than when fixating on outer areas (due to lens defects). 

By knowing a user's visual field, one can optimize the rendering pipeline to skip areas that are not seen. This will enable faster frame rates (up to 2x performance gains) and lower perceived latency, and therefore a more immersive VR experience.